exasol_transformers_extension/udfs/models/model_downloader_udf.py

```diff
@@ -26,13 +26,11 @@ def __init__(self,
                  base_model_factory: ModelFactoryProtocol = transformers.AutoModel,
                  tokenizer_factory: ModelFactoryProtocol = transformers.AutoTokenizer,
                  huggingface_hub_bucketfs_model_transfer: HuggingFaceHubBucketFSModelTransferSPFactory =
-                 HuggingFaceHubBucketFSModelTransferSPFactory(),
-                 bucketfs_factory: BucketFSFactory = BucketFSFactory()):
+                 HuggingFaceHubBucketFSModelTransferSPFactory()):
```

Comment:
Not a fan of this repetitive java style long names, imho they don't provide any value.

Reply:
Me neither.